lms/djangoapps/courseware/toggles.py

```diff
@@ -115,3 +132,10 @@ def courseware_mfe_first_section_celebration_is_active(course_key):
         COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES.is_enabled(course_key) and
         COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES_FIRST_SECTION_CELEBRATION.is_enabled(course_key)
     )
+
+def courseware_mfe_first_discussion_celebration_is_active(course_key):
+    return (
+        REDIRECT_TO_COURSEWARE_MICROFRONTEND.is_enabled(course_key) and
+        COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES.is_enabled(course_key) and
```

Comment:
Do we still need this one? Like, we're past the point of wanting to turn all progress milestones off yah? Unless this is like the redirect-to-courseware flag, and we're keeping it just to be able to opt out some courses?

Reply:
Yeah, I'm not sure. Might be a good discussion for the team, but I figured it can't hurt continuing to have it in case we do want to use it. And then if we want to remove, it's not a big lift to get it out along with the `courseware_mfe_first_section_celebration_is_active`